.github/workflows/publish-nuget.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish NuGet Packages
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  build-and-publish:
+    name: Build and Publish NuGet
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v3
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: 9.0
+    - name: Extract Version from Tag
+      run: echo "VERSION=${GITHUB_REF##*/}" >> $GITHUB_ENV
+    - name: Install Dependencies
+      run: dotnet restore
+    - name: Build Solution
+      run: |
+        dotnet build --configuration Release \
```

Comment:
I ran both commands locally for a sanity check ([NuGet Package Explorer](https://github.com/NuGetPackageExplorer/NuGetPackageExplorer) is awesome for that, by the way) and it packages as $version, but the assembly version is still 1.0.0.0. And that's partly the fault of our `Packages.props` I guess... but we'll fix that another day.

I manged to build it correctly by doing this:

```sh
dotnet build --no-restore -c Release -p:CI_BUILD=true -p:Version=$version -p:PackageVersion=$version
dotent pack --no-build -c Release -p:CI_BUILD=true -p:Version=$version -p:PackageVersion=$version -o artifacts/
```

Reply:
Damn, I checked the individual outputs of the two steps, but not the assembly version of the package's unpacked DLL. I checked now and could see it still showing `1.0.0.0`.

Reply:
I'll trust the package explorer.. it should work, we can always create a preview-release and check that one before fully releasing it